js/index.js

```diff
@@ -34,8 +35,8 @@ var typeMap = ['','','','','','','DE','IND'];
 $(document).ready(function() {
 
 	$.getJSON( "manifest.webapp", function( json ) {
-
-		getServerInfo(dhisPath, function(info) {
+                baseWebApp = json.launchUrl.substring(0,json.launchUrl.indexOf("/api/"));
```

Comment:
This should work, but there is another way, probably more robust: we can change `manifest.webapp` to like this (see [docs](https://docs.dhis2.org/2.25/en/developer/html/apps_creating_apps.html)):

```
"activities": {
        "dhis": {
            "href": "*",
            "namespace": "datadictionary"
        }
    },
```

And then simply: `var baseWebApp = json.activities.dhis.href;`

What do you think?